notifications.bs

```diff
@@ -70,7 +70,7 @@ run.
 
 <p>A <a>notification</a> has an associated
 <dfn for=notification id=silent-preference-flag>silent preference flag</dfn> which is initially
-unset. When set indicates that no sounds or vibrations should be made.
```

Comment:
I kept this in because the user agent could still suppress any default sounds the platform's notification center may make, which feels appropriate for a property called `silent`.

Reply:
Ah, good call.